### ISSUE TITLE
New User Enhancement: Disable multi-session loading by default

### ISSUE THREAD

**Issue by FlameSoulis** [Yellow-Dog-Man/Resonite-Issues#538]
### Is your feature request related to a problem? Please describe.

This one is going to be controversial, but I'm going to be blunt: new people coming in have no clue that they are in multiple sessions at the same time, and due to the laggy nature of the default cloud home, many user's performance numbers are terrible by not being aware of existing in multiple sessions at the same time!

While this is a handy feature for advanced users, a new user will not immediately get this, claim the platform is laggy, and leave. Additionally, getting notified about 'sessions crashing' is very confusing to those who aren't aware.

### Describe the solution you'd like

Create a new setting that allows the option to use multiple sessions and have it disabled by default. This means the feature remains available for those who are used to it and disabled for newer users as to avoid causing a bad experience.

### Describe alternatives you've considered

Perhaps a first-time message when visiting the worlds tab to quickly explain active sessions, though it'd potentially require addressing any issues that involve active sessions the user can focus on, such as cases where lag can prevent them from showing.

### Additional Context

While this may need to be made into a separate issue, perhaps having the connected sessions show how much resources they are demanding could help even advanced users determine the laggy culprit, as at this time, some sessions/worlds just eat so many resources, their active nature can mean the difference from 50FPS while in VR and a full 144 on an Index.

**Comment by epicEaston197** [comment]
My main concern with this is users have created tools that would unfocus you from the current world that you're in having multi-session loading turned off by default would result in those harmless items becoming soft moderation tools which was not their intention

**Comment by FlameSoulis** [comment]
How is being unfocused from a world harmless? To a new user, you made it seem they were just disconnected from the session they were in, and unless they can figure out how to focus back on it in the Worlds tab, it sounds like you basically "hacked" them.

This also goes for exit strat items as well: It's a powerful function, but view it from the aspect of a new user... it's pretty bad. Cool, yes, but also... not great.

Additionally, you missed the point of it being an option to still have multiple sessions, so this doesn't affect more experienced users.

**Comment by Crusher50** [comment]
I think with the implantation of #396 this won't be as big of an issue. That plus the tutorial teaching new users about this functionality will also help.

Personally, I don't believe it should be disabled by default. However, an option to disable it would be beneficial to users that don't want that functionality.

**Comment by Zyro1331** [comment]
> I think with the implantation of #396 this won't be as big of an issue.

I agree with Stella here. Having a feature that makes multi-world presence much more visible to the end user is a much better idea than completely disabling this core feature due to the end user being new is a lot more limiting when it's not necessary. 

Making this feature's presence more similar to a Web browser's tabs by placing an interface at the top or side of your Dash in the form of a facet would make it much easier to grasp to the end user. 
@ProbablePrime made a facet previously that would show you the worlds you were in as a Vertically stacked list, and I find that it's so useful that it should become a default feature rather than using the Contacts list to see where you are. 
If it can be modernized with the current UI design and made default, I feel that it would be a whole lot easier to figure out things yourself. 

Also, another thing to note, since multi-world presence is a very core feature, this could make the Local Home and Cloud Home difficult to keep open consistently without completely reloading the world every time the session you are in closes or the world crashes, alongside having to implement the detection for those core worlds to keep open. So it would simply just be easier to implement the above facet. It would also allow users to simply give themselves performance just hitting the "X" button on unfocused worlds as they would when their Web browser is running slow.

**Comment by FlameSoulis** [comment]
As outside discussions have more or less agreed, it's a lack of presentation (inability to understand the concept of multiple sessions easily) and information (lack of distributed knowledge of the feature).

Reviewing #396 and #539 both confirm this, and they do get my vote. Examining this concept further, it presents some additional issues, such as "if a user is in multiple sessions, then turns the feature off... what should be done?" In some regards, it opens the door to potentially more issues rather than just addressing the issues that created the desire for a solution to the problem.

I'll keep it open for a smidge longer, in case other comments regarding this comes up, but I think the mentioned issues deserve more attention.

**Comment by AnotherFoxGuy** [comment]
As a casual user, the amount of times I've used the multi-session feature has been 0 times so far 
When I join a world currently I just close the other sessions manually, so having a setting to make it single-session would be nice

**Comment by JackTheFoxOtter** [comment]
I believe one of the main issues is that it's just not very intuitive to use. Instead of disabling it outright per default, restructuring the world browser might help a bit with that. For example, I think having a separate list above the main world grid with "Sessions you are currently in" and a big "Leave/Close" button would go a long way, because currently they don't show up if a filter is active that otherwise wouldn't show the world.

**Comment by Banane9** [comment]
> As a casual user, the amount of times I've used the multi-session feature has been 0 times so far

Even as an experienced user, the time I've spent in multiple sessions at once is limited to transitioning from one to the other usually, as a heavy world in the background can easily cost you 20-30 FPS in the focused world too


> For example, I think having a separate list above the main world grid with "Sessions you are currently in" and a big "Leave/Close" button would go a long way, because currently they don't show up if a filter is active that otherwise wouldn't show the world.

Agreed
